web/src/testUtils.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+
+/**
+ * Render a component which expects to be inside a router
+ */
+export function renderRoute(component: React.ReactNode) {
```

Comment:
While `renderWithBrowserRouter` is quite a lot longer, I'd suggest to use that naming to make it clearer on what it does. What do you think?

Reply:
Then maybe the comment might not even be needed anymore.

Reply:
> Did you stumble upon something else that could make contributions easier? I've already merged your other PR regarding the wrong callback in the README. Anything else that would have made your contribution easier?

Not really (apart from my default editor formatting not matching the project style). While there was a couple extra steps compared to common-voice, the setup was really painless and worked out of the box :)